_episodes/08-putting-it-all-together.md

```diff
@@ -86,209 +89,226 @@ also change the name of the columns in the DataFrame like this:
 
 ~~~
-    import matplotlib.pyplot as plt
+import pandas as pd
+surveys = pd.read_csv("data/surveys.csv")
+my_plot = surveys.plot("hindfoot_length", "weight", kind="scatter")
+my_plot
 ~~~
 {: .language-python}
 
-By default, matplotlib will create the figure in a separate window. When using
-ipython notebooks, we can make figures appear in-line within the notebook by
-writing:
+![Scatter plot of survey data set](../fig/08_scatter_surveys.png)
 
-~~~
-    %matplotlib inline
-~~~
-{: .language-python}
+The returned object is a matplotlib object (check it yourself with `type(my_plot)`) and the power of matplotlib is available to further adjust these plots as it is created with matplotlib itself.
 
-We can start by plotting the values of a list of numbers (matplotlib can handle
-many types of numeric data, including numpy arrays and pandas DataFrames - we
-are just using a list as an example!):
+> ## Tip
+> Matplotlib itself can be overwhelming, so a useful strategy is to 
+> do as much as you easily can in a convenience layer, i.e. start
+> creating the plot in Pandas or plotnine, and then use matplotlib
+> for the rest.
+{: .callout}
 
-~~~
-    list_numbers = [1.5, 4, 2.2, 5.7]
-    plt.plot(list_numbers)
-    plt.show()
-~~~
-{: .language-python}
+We will cover a few basic commands for creating and formatting plots with matplotlib in this lesson. A great resource for help creating and styling your figures is the matplotlib gallery
+(<http://matplotlib.org/gallery.html>), which includes plots in many different
+styles and the source code that creates them. 
 
-The command `plt.show()` prompts Python to display the figure. Without it, it
-creates an object in memory but doesn't produce a visible plot. The ipython
-notebooks (if using `%matplotlib inline`) will automatically show you the figure
-even if you don't write `plt.show()`, but get in the habit of including this
-command!
+### Using the matplotlib package
 
-If you provide the `plot()` function with only one list of numbers, it assumes
-that it is a sequence of y-values and plots them against their index (the first
-value in the list is plotted at `x=0`, the second at `x=1`, etc). If the
-function `plot()` receives two lists, it assumes the first one is the x-values
-and the second the y-values. The line connecting the points will follow the list
-in order:
+First, import the pyplot toolbox:
 
 ~~~
-    plt.plot([6.8, 4.3, 3.2, 8.1], list_numbers)
-    plt.show()
+import matplotlib.pyplot as plt
 ~~~
 {: .language-python}
 
-A third, optional argument in `plot()` is a string of characters that indicates
-the line type and color for the plot. The default value is a continuous blue
-line. For example, we can make the line red (`'r'`), with circles at every data
-point (`'o'`), and a dot-dash pattern (`'-.'`). Look through the matplotlib
-gallery for more examples.
+By default, matplotlib creates a figure in a separate window. When using
```

Comment:
@wrightaprilm, I think this note about `%matplotlib inline` should be a callout box and introduced after the first plot is created.